apps/survey/app/src/math-utils.js

```diff
@@ -49,21 +49,31 @@ function highestValueIndex(values) {
     .sort((v1, v2) => v2.value.minus(v1.value))[0].index
 }
 
-export function scaleBigNumberValuesSet(values, total = new BigNumber(100)) {
+// Scale a set of 0 => 1 values to equal `total`.
```

Comment:
Maybe `[0, 1] values` would work better as math notation? Or even a set of values summing to 1?